www/go/core/jmap/Router.php

```diff
@@ -58,13 +58,15 @@ public function error($type, $status, $detail) {
 	public function run() {	
 
 		$body = Request::get()->getBody();
+
```

Comment:
Can we remove these modified files that only have blank lines in them? No need to include them in the PR if there's no code changes.